ocp_resources/trustyai_service.py

```diff
@@ -0,0 +1,59 @@
+from ocp_resources.resource import NamespacedResource
+from typing import Optional, Any, Dict
+
+
+class TrustyAIService(NamespacedResource):
+    """
+    https://github.com/trustyai-explainability/trustyai-service-operator/blob/main/api/v1alpha1/trustyaiservice_types.go
+    """
+
+    api_group: str = NamespacedResource.ApiGroup.TRUSTYAI_OPENDATAHUB_IO
+
+    def __init__(
+        self,
+        storage: Dict[str, Any],
+        data: Dict[str, Any],
+        metrics: Dict[str, Any],
+        replicas: Optional[int] = 1,
```

Comment:
Please do not set any defaults, if the user sent we set it, else we let k8s to handle it and set the default.